zone/client.h

```diff
@@ -1003,8 +1005,29 @@ class Client : public Mob
 	void EnteringMessages(Client* client);
 	void SendRules(Client* client);
 
+	uint32 m_time_since_last_position_check;
```

Comment:
Please put a comment above this block of function calls just informing the next developer this section of defines are all related to cheating or punk busting 

```cpp
// cheat detection
```